homeassistant/components/sensor/discogs.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_TOKEN)
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:album'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_TOKEN): cv.string,
+})
+
+REQUIREMENTS = ['discogs_client==2.2.1']
+
+SCAN_INTERVAL = timedelta(seconds=7200)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up the discogs sensor."""
+    token = config.get(CONF_TOKEN)
+
+    async_add_devices([DiscogsCollection(token)], True)
+    return True
+
+
+class DiscogsCollection(Entity):
+    """Get a user's number of records in collection."""
+
+    def __init__(self, token):
+        """Initialize the sensor."""
+        import discogs_client
+        discogs = discogs_client.Client(
+            'HomeAssistantDiscogs/0.1.0', user_token=token)
```

Comment:
Can you please use `SERVER_SOFTWARE` (`from homeassistant.helpers.aiohttp_client import SERVER_SOFTWARE`)?

Reply:
Instead of the 'HomeAssistantDiscogs' string?